docs/docs/intro.md

```diff
@@ -1,23 +1,173 @@
 ---
-title: Welcome
+title: QuickStart
 sidebar_position: 1
 ---
 
-# Welcome to Warp 2.0
+<!-- # Welcome to Warp 2.0
 
 We are excited to announce the second version of Warp, now designed to transpile your Solidity code directly into Cairo.
 
 Warp 1 set out to show that compiling Solidity code into Cairo was possible, and paved the way for developers to access the benefits of Starknet without needing to master Cairo. Using everything we learned from Warp 1, we have written a new version adding vast improvements to contract efficiency and user experience. In this blog, we will talk through the improvements made to Warp, transpile OpenZeppelin’s ERC20 contract, and describe future plans for the project.
 
-### Warp 2 vs Warp 1
+#### Want to know more on Warp 2.0? Read the blog [here](https://medium.com/nethermind-eth/warp-2-0-transpiling-directly-from-solidity-to-cairo-9bf41a6d26ee). -->
```

Comment:
Let's add a short introduction before Quickstart:
```
# Welcome to Warp!

Warp is a transpiler from Solidity to Cairo. It allows you to deploy your Solidity contracts on Starknet.
```

Reply:
Hey @piwonskp , I made the required changes. Please review it and let me know if any more changes are required.

![image](https://github.com/NethermindEth/warp/assets/101472765/1af5e3e5-df42-47c7-b4a5-6511b3d8bf80)